config/global.ini.php

```diff
@@ -639,6 +639,7 @@
 ; By enabling this flag we will for example not allow the installation of a plugin via the UI as a plugin would be only
 ; installed on one server or a config one change would be only made on one server instead of all servers.
 ; This flag doesn't need to be enabled when the config file is on a shared filesystem such as NFS or EFS.
+; During maintenance mode, multiple environment will continue return 200 as response on API request.
```

Comment:
Not sure if that makes it clear enough. Actually this is not limited on API requests. I would suggest something like `When enabled, Matomo will return the response code 200 instead of 503 in maintenance mode.`
Or maybe @justinvelluppillai has a better idea here.